hyperspy/drawing/mpl_hse.py

```diff
@@ -83,7 +83,7 @@ def plot_signal(self):
             self.xlabel += ' (%s)' % self.axes_manager.signal_axes[0].units
         self.ylabel = 'Intensity'
         self.axis = self.axes_manager.signal_axes[0].axis
-        sf = spectrum.SpectrumFigure(title=self.signal_title +
+        sf = spectrum.Signal1DFigure(title=self.signal_title +
```

Comment:
`spectrum.SpectrumFigure` probably needs renaming -> `signal1d.SpectrumFigure`?

The same goes for the equivalent image module.